ttsfm/cli.py

```
@@ -0,0 +1,362 @@
+#!/usr/bin/env python3
+"""
+Command-line interface for TTSFM.
+
+This module provides a command-line interface for the TTSFM package,
+allowing users to generate speech from text using various options.
+"""
+
+import argparse
+import sys
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)